examples/miscellaneous/plot_outlier_detection_bench.py

```diff
@@ -17,181 +19,361 @@
 """
 
 # Author: Pharuj Rajborirug <pharuj.ra@kmitl.ac.th>
+#         Arturo Amor <david-arturo.amor-quiroz@inria.fr>
 # License: BSD 3 clause
 
-print(__doc__)
+# %%
+# Datasets preprocessing and model training
```

Comment:
```suggestion
# Dataset preprocessing and model training
```
Even though there are several datasets it reads nicer like this. But don't ask me what the grammar rule book says about which one is correct ;)